proxy/src/coco/control.rs

```diff
@@ -0,0 +1,148 @@
+use std::env;
+
+use librad::meta::entity;
+use librad::meta::project;
+use librad::net::peer::PeerApi;
+use radicle_surf::vcs::git::git2;
+
+use crate::coco::peer::{init_project, User};
+use crate::error;
+
+/// This function exists as a standalone because the logic does not play well with async in
+/// `replicate_platinum`.
+fn clone_platinum(
```

Comment:
This should go in source as the original TODO mentioned as it only operates on repos on disk.

Reply:
Ya, but it's only used in `replicate_platinum` so I thought that it living beside its `pub fn` would make more sense.